Purchases/Public/Purchases.swift

```diff
@@ -133,12 +136,54 @@ public typealias DeferredPromotionalPurchaseBlock = (@escaping PurchaseCompleted
      * Set a custom log handler for redirecting logs to your own logging system.
      * By default, this sends Info, Warn, and Error messages. If you wish to receive Debug level messages,
      * you must enable debug logs.
+     *
+     * - Note:``verboseLogHandler`` provides additional information.
+     *
+     * - Seealso: ``verboseLogHandler``
+     * - Seealso: ``logLevel``
      */
-    @objc public static var logHandler: (LogLevel, String) -> Void {
+    @objc public static var logHandler: LogHandler {
```

Comment:
The existing API is unmodified, and it forwards to the new one, so it's backwards compatible. Eventually we could deprecate this.